src/message/MessageListWeb.js

```diff
@@ -56,6 +57,21 @@ export default class MessageListWeb extends Component<Props> {
   };
 
   componentWillReceiveProps = (nextProps: Props) => {
+    if (this.props.fetchingOlder !== nextProps.fetchingOlder) {
```

Comment:
Send one type of message, pass both variables.
It is entirely possible that both values have changed.